### PR TITLE
9.0 add date range Sorrento Delivery

### PR DESCRIPTION
**Pull request by lmignon**
A new addon to manage date range into odoo https://docs.google.com/document/d/1_czJry1KcaXKCXZr72C7WEHq3d05P2gQEEenbqToBak/edit#heading=h.rmvj12bidc97
The ranges are proposed as filter in all tree view on date or datetime fields

### CODE REVIEW

File: date_range/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Date Range
+==========
+
+This module lets you define global date ranges that can be used to filter
+your values in tree views.
+
+Usage
+=====
+
+To use this module, you need to:
```
Comment (tremlin) [inline date_range/README.rst]: We should a brief explanation here.

File: date_range/tests/test_date_range.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(S)> <AUTHOR(S)>
```
Comment (tremlin) [inline date_range/tests/test_date_range.py]: Copyright header is missing here.

File: date_range/models/date_range.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            if res:
+                dt = self.browse(res[0][0])
+                raise ValidationError(
+                    _("%s overlaps %s") % (this.name, dt.name))
+
+    @api.multi
+    def get_domain(self, field_name):
+        self.ensure_one()
+        return [(field_name, '>=', self.date_start),
+                (field_name, '<', self.date_end)]
```
Comment (tremlin) [inline date_range/models/date_range.py]: Is it on purpose that the end date is _not_ included in the range?

File: date_range/README.rst (hunk 1 above)
Comment (eLBati) [inline date_range/README.rst]: Or remove these lines

**Comment by eLBati** [comment]
tested and :+1:

File: .travis.yml [hunk 4]
```diff
@@ -6,6 +6,7 @@ python:
   - "2.7"
 
 addons:
+  postgresql: "9.2" # minimal postgresql version for the daterange method
```
Comment (lepistone) [inline .travis.yml:9]: Out of curiosity, is the default is less than this?

**Comment by lepistone** [comment]
great, thanks! 👍

**Comment by adrienpeiffer** [comment]
:+1:

File: .travis.yml (hunk 4 above)
Comment (lmignon) [inline .travis.yml:9]: @lepistone yes the default is 9.1

**Comment by jgrandguillaume** [comment]
Thank for this great module !

:+1:

**Pull request merged**